modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -849,10 +898,22 @@ const Fields = ({
 		});
 
 	const onEditFDSField = ({editedFDSField}: {editedFDSField: IFDSField}) => {
+		let updatedFDSField: IFDSField;
```

Comment:
Same case here. Saving a field without translation will display an empty string.
Maybe we could validate that the current user language is not empty.

Reply:
Or pre-populate with field `name`? It's a good idea for a future improvement! We can return to this after PR is merged.